modules/dkan/dkan_harvest/dkan_harvest.module

```diff
@@ -1460,3 +1460,8 @@ function dkan_harvest_delete_confirmation_submit($form, $form_state) {
 
   dkan_harvest_process_source_datasets($selected_sources, $datasets_op);
 }
+
+function validateDate($date, $format = 'Y-m-d') {
```

Comment:
CamelCase functions don't follow Drupal naming convention. Should be dkan_harvest_validate_date(). Also, I think this function should live in dkan_dataset, since it's used by that module as well.